### PR TITLE
1713 no mint for you

### LINKED ISSUE #1713: Unable to mint signular based NFTs
### What happened?

I tried to mint the NFT to my old collection I have made many months ago 

```graphql
query bigOne {
  collection_i_want: collectionEntity(id: "8CC1B91E899D9BE40E-2CENTS") {
    id
    name
  }
  
  collection_list_i_have: collectionEntities(filter: { issuer: {equalTo: "Fksmad33PFxhrQXNYPPJozgWrv82zuFLvXK7Rh8m1xQhe98" } max: { equalTo: 0 } }) {
    nodes {
      id
      name
      max
    }
  }
}
```
The result

```json
{
  "data": {
    "collection_i_want": null,
    "collection_list_i_have": {
      "nodes": [
        {
          "id": "8cc1b91e899d9be40e-2CENTS",
          "name": "2 cents",
          "max": 0
        },
        {
          "id": "8cc1b91e899d9be40e-VIKTORY",
          "name": "Viktory",
          "max": 0
        }
      ]
    }
  }
}
```

looks like there is a bug in both `legacy` and `beta`

### Please reproduce in steps

-

### Expected Behavior

-

### What browsers are you seeing the problem on?

_No response_

### Are you logged in?

_No response_

### At which address did you encounter bug?

_No response_

### Screenshots

```json
{
  "interaction": "MINTNFT",
  "value": "{\"events\":[],\"name\":\"K. Outline\",\"instance\":\"K._OUTLINE\",\"transferable\":1,\"collection\":\"8CC1B91E899D9BE40E-2CENTS\",\"sn\":\"0000000000000002\",\"_id\":\"\",\"id\":\"\",\"metadata\":\"ipfs://ipfs/QmYnQoQTEaTSyK8QV77wfWjE1MHp1XSSvYxCBzDNGGGyTn\",\"currentOwner\":\"Fksmad33PFxhrQXNYPPJozgWrv82zuFLvXK7Rh8m1xQhe98\"}",
  "reason": "[CONSOLIDATE canOrElseError] Callback not exists"
}
```

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by vikiival**
- :bug: NFTUtils now use collectionID instead of symbol
- :bug: correct resolution of collectionId
- :art: #363636 is $grey-darker

Thank you for your contribution to the KodaDot NFT gallery, 
we really appreciate your contribution!

### PR type
- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### Before submitting this PR, please make sure:
- [x] Your code builds clean without any erros or warnigns
- [x] You've posted screenshot of demonstrated change in this PR
- [x] Merged recent default branch, **main** and you have no conflicts
- [ ] Didn't break any original functionality 

### Optional
- [ ] You've tested it on mobile
- [ ] Are there any edge cases? Name if any 

### What's new? (may be part of changelog)
- Close #1713

### CODE REVIEW

**Review by roiLeo — COMMENTED** [review]
to be able to create a nft it must be in a collection?

**Comment by vikiival** [comment]
Yup

**Review by roiLeo — CHANGES_REQUESTED** [review]
Screenshot 2022-01-07 at 16-48-49 KodaDot - Polkadot Kusama NFT explorer

**Comment by vikiival** [comment]
271490047_10159715089384630_2808985056504806907_n

**Comment by vikiival** [comment]
:bug: text on landing page became dark

**Review by roiLeo — APPROVED** [review]
What will happen if I go to create > simple & choose NFT tab, will it generate a collection id automatically?

File: components/rmrk/service/NftUtils.ts [hunk 1]
```diff
@@ -136,7 +134,7 @@ class NFTUtils {
 
   public static generateRemarks(mint: SimpleNFT, caller: string, version = '1.0.0', encode?: boolean, updateName?: UpdateFunction): MintType | string[] {
     const collection = NFTUtils.createCollection(caller, mint.symbol, mint.name, mint.metadata, mint.max, version)
-    const nfts = Array(mint.max).fill(null).map((e, i) => NFTUtils.createNFT(caller, i, mint.symbol, updateName ? updateName(mint.name, i) : mint.name, mint.metadata))
+    const nfts = Array(mint.max).fill(null).map((e, i) => NFTUtils.createNFT(caller, i, collection.id, updateName ? updateName(mint.name, i) : mint.name, mint.metadata))
```
Comment (vikiival) [inline components/rmrk/service/NftUtils.ts:137]: CC @roiLeo

Comment (roiLeo) [reply components/rmrk/service/NftUtils.ts:137]: alright then

**Pull request merged**